src/get-started/localisation/index.md.njk

```diff
@@ -0,0 +1,101 @@
+---
+layout: layout-pane.njk
+title: Using localisation with GOV.UK Frontend
+section: Get started
+theme: How to guides
+order: 5
+description: Localisation allows you to translate GOV.UK Frontend components into other languages
+---
+
+Localisation allows you to translate GOV.UK Frontend components into other languages
+
+## Set up localisation using Nunjucks
+
+If you’re using the Nunjucks macros provided by GOV.UK Frontend, you can localise the UI text that appears within components by passing translation strings as named parameters.
+
+The individual components using these parameters include:
```

Comment:
It's a mixture. Almost every component is already 'localisable' (insomuch as the content in them is configurable), so it's kind of like this. 

I'm not really sure how best to convey this. It might be better to say these are the components we have specific localisation guidance for, rather than saying they're the only ones that are localisable? 

JavaScript changes only:
- Accordion

Nunjucks changes only: 
- Footer
- Header

JavaScript and Nunjucks changes: 
- Character count

Guidance changes only: 
- Error message
- Summary list

Reply:
```suggestion
We have provided specific localisation guidance for some of the components. These include:
```